Dockerfile

```diff
@@ -16,6 +32,11 @@ RUN pip-sync requirements.txt
 ADD . /usr/src/code
 WORKDIR /usr/src/code
 
+COPY package.json /usr/src/code/package.json
```

Comment:
put this block of node and yarn code above line 32 to enable docker build cache to do it's thing